src/Workspaces/Core/Portable/FlowAnalysis/SymbolUsageAnalysis/SymbolUsageAnalysis.DataFlowAnalyzer.FlowGraphAnalysisData.cs

```diff
@@ -195,6 +189,100 @@ private PooledHashSet<(ISymbol, IOperation)> GetOrCreateSymbolWritesInBlockRange
                     return writesInBlockRange;
                 }
 
+                private void AddOperationsInRange(
+                    ControlFlowGraph cfg,
+                    int firstBlockOrdinal,
+                    int lastBlockOrdinal,
+                    PooledHashSet<IOperation> operationsBuilder)
+                {
+                    // Compute all descendant operations in basic block range.
+                    for (int i = firstBlockOrdinal; i <= lastBlockOrdinal; i++)
```

Comment:
do these need cancellation token? asking since loop seems could be big?

Reply:
Sure, I can thread in the cancellation token through the dataflow walker API